src/main/java/net/dries007/tfc/world/region/RegionGenerator.java

```diff
@@ -87,6 +93,12 @@ public RegionGenerator(Settings settings, RandomSource random)
                 .spread(0.15f)
                 .scaled(-80f, 40f)); // Bias slightly negative, as we bias near-ocean areas to be positive rainfall, so this encourages deserts inland
 
+        this.rainfallVarianceNoise = simpleBaseNoise()
+            .add(new OpenSimplex2D(random.nextInt())
+                .octaves(2)
+                .spread(0.3f)
+                .scaled(-.2f, 0.2f));
```

Comment:
```suggestion
        this.rainfallVarianceNoise = new OpenSimplex2D(random.nextInt())
            .octaves(2)
            .spread(0.3f)
            .scaled(-.2f, 0.2f);
```
No need to do the simple base noise and add, can just use the simplex noise directly.